runner.py

```diff
@@ -0,0 +1,495 @@
+#!/usr/bin/env python
```

Comment:
This should be `python3`.

Reply:
This should be fixed in the latest's commit.